src/server/utils/getCashaySchema.js

```diff
@@ -3,4 +3,6 @@ require('babel-polyfill');
 const {transformSchema} = require('cashay');
 const graphql = require('graphql').graphql;
 const rootSchema = require('../graphql/rootSchema');
+const r = require('../database/rethinkDriver');
```

Comment:
I'll think about it post merge…


Sent with Mixmax





On Wed, Jun 15, 2016 6:55 PM, Matt Krick notifications@github.com wrote:
In src/server/utils/getCashaySchema.js :

 
 
 
 
 


we can comment out the const r... like we do the line below it. perhaps we should think of a way to use
cashay-loader inside the webpack config itself?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub , or mute the thread .



Jordan @jrhusney / 612.227.5673